src/Dom/Document.php

```diff
@@ -55,71 +62,6 @@ final class Document extends DOMDocument
      */
     const HTML_DOCTYPE_REGEX_PATTERN = '#<!doctype\s+html[^>]+?>#si';
 
-    /**
```

Comment:
FYI: The removals here necessitate changes to the AMP plugin. See https://github.com/ampproject/amp-wp/pull/6498.